### ISSUE TITLE
DropDown list is not visible

### ISSUE THREAD

**Issue by aniketparab4** [toystars/react-native-multiple-select#19]
Please can you help me out with my issue I have posted the issue on this link https://stackoverflow.com/questions/46646491/multiselect-in-react-native. I am trying to implement  your component, thank you for sharing this amazing component. but I got some problem implementing it.
 Actually i am fetching the data from API in the JSON format, my data would be somewhat like this [{"id":"14","name":"Ondo"},{"id":"15","name":"Ogu‌​n"},{"id":"16","name‌​": "Calabar"}]
and am saving this data to state. And the state is passed to the Item prop of multiselect. But i am not able to get the list of name in dropdown it shows the message as "No item to display". According to your doc items should contain the array of object which is the same i am passing to it. but still unable to view the list.

**Comment by toystars** [comment]
@aniketparab4 have you been able to get this fixed?

**Comment by toystars** [comment]
@aniketparab4 I please check the Stackoverflow page, I dropped a comment there.

**Comment by aniketparab4** [comment]
No. the issue is not fixed. I have made the changes on stackoverflow...!!
The question is how do I fill the dropdown list by fetching the data from API.

**Comment by aniketparab4** [comment]
how do i display the name in dropdown list?? as the data is fetch from API in JSON format !!

**Comment by abhi-nahi** [comment]
Any solutions or alternates???

**Comment by AugustoAleGon** [comment]
@aniketparab4 can you share your repo? I already read your post. But I am not sure how the data comes. The multiselect components works but I think that you are passing a string and not and object to the component.

**Comment by AugustoAleGon** [comment]
@abhi-nahi did you check the example on the link? https://github.com/AugustoAleGon/react-native-multiple-select-sample this version
